src/BoilerPlate/App.cpp

```diff
@@ -144,31 +260,28 @@ namespace Engine
 
 	void App::Render()
 	{
-		//            r      g    y		b
-		glClearColor(0.0, 0.50f, 0.50f, 1.0f);
-		glClear(GL_COLOR_BUFFER_BIT);
+		//            r      g    b		a
+		glClearColor(0.10f, 0.15f, 0.40f, 1.0f);
```

Comment:
You should have a palette and a color class... What's the need for this kind of comment?